src/platform/plugins/shared/dashboard/public/dashboard_api/layout_manager/layout_manager.ts

```diff
@@ -282,14 +283,21 @@ export function initializeLayoutManager(
 
     layout$.next(await placeNewPanel(uuid, panelPackage, grid, options?.beside));
 
-    if (options?.displaySuccessMessage) {
+    const { scrollToPanel, displaySuccessMessage } = {
+      scrollToPanel: true,
+      displaySuccessMessage: false,
+      ...options,
+    };
+    if (displaySuccessMessage) {
       const title = (serializedState?.rawState as SerializedTitles)?.title;
       coreServices.notifications.toasts.addSuccess({
         title: getPanelAddedSuccessString(title),
         'data-test-subj': 'addEmbeddableToDashboardSuccess',
       });
     }
-    trackPanel.setScrollToPanelId(uuid);
+    if (scrollToPanel) {
+      trackPanel.setScrollToPanelId(uuid);
+    }
```

Comment:
The screen would jitter when adding multiple panels. So we need the ability to turn it off when we know we are adding more than one panel in sequence.